Tests/Pcap++Test/Makefile

```diff
@@ -70,7 +73,11 @@ dependents:
 	
 Pcap++Test: start create-directories $(OBJS_FILENAMES)
 	@cd $(MEMPLUMBER_HOME) && $(MAKE) memplumber_sources
+ifdef FREEBSD
+	@$(CXX) $(COMMON_DEBUG_LIB_DIR) $(PCAPPP_LIBS_DIR) $(PCAPPP_BUILD_FLAGS) -lexecinfo -o "./Bin/Pcap++Test$(BIN_EXT)" $(OBJS_FILENAMES) $(MEMPLUMBER_OBJS_FILENAMES) $(PCAPPP_LIBS)
+else

```

Comment:
wouldn't it be better to add `-lexecinfo` to `PcapPlusPlus.mk.freebsd`?

Reply:
Done. See my commit 3d8f2894b2.